STEERING-COMMITTEE.md

```diff
@@ -236,19 +244,20 @@ found in the [elections folder](./elections).
 
 ### Limitations on Company Representation
 
-No more than two seats may be held by employees of the same organization (or
-conglomerate, in the case of companies owning each other). Additionally, each
-end user seat (when available) must belong to an employee from a different
-organization. If the results of an election result in greater than two employees
-of the same organization, the lowest vote getters from any particular employer
-will be removed until representation on the committee is down to two.
-
-If employers change because of job changes, acquisitions, or other events, in a
-way that would yield more than 2 seats being held by employees of the same
-organization, sufficient members of the committee must resign until only two
-employees of the same employer are left. If it is impossible to find sufficient
-members to resign, all employees of that organization will be removed and new
-special elections held. In the event of a question of company membership (for
+No more than two contributing seats may be held by employees of the same organization
+(or conglomerate, in the case of companies owning each other). Additionally, each end
+user seat (when available) must belong to an employee from a different
```

Comment:
Is the end user seat optional or mandatory? Maybe we should skip "(when available)" or otherwise mention that an end user seat is optional (which then also would decrease the size of the SC)

Reply:
Removed, thanks!